hudi-spark-datasource/hudi-spark/pom.xml

```diff
@@ -56,6 +56,10 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
+          <configuration>
+            <source>1.8</source>
```

Comment:
Good catch! Should we propagate this configs to every module?

Let's also parameterize actual version so it's easier to change from the main pom

Reply:
These changes were not useful in fixing the issue. Removed it from the latest commit.

Reply:
This configuration has already been defined in the base pom.